schema-dex-amm-extended.graphql

```diff
@@ -768,15 +1060,39 @@ type PositionSnapshot @entity {
   " Position of this snapshot "
   position: Position!
 
-  " Input token balances in this position, in native amounts "
-  inputTokenBalances: [BigInt!]!
+  " Type of token used to track liquidity "
+  liquidityTokenType: TokenType
+
+  ## total position liquidity
```

Comment:
Quotes, there are some more below as well

Reply:
There are more elsewhere too

Reply:
Okay - I believe I got all of them.